lore/admins/reboot-zuul.md

```diff
@@ -0,0 +1,82 @@
+---
+name: rebooting zuul
+layout: default
+---
+
+# Rebooting Zuul
+
+## Overview
+
+Periodically, Zuul will need to be restarted, whether to pick up new changes or even routine system maintenance.  To do so, you will first need to connect to the zuul node(s) by using the bastion as a jump host.  Currently, one would need to access the nodepool instance in the same manner in order to kill the current running images.
+
+## Connect to zuul node and restart
+
+From the bastion, use the zuul shortcut to connect.
+
+```text
+user@bastion$ zuul
```

Comment:
Most of our lore has commands listed like this:

```markdown
    ```shell
    $ zuul
    ```
```

But, since we are running commands on multiple hosts, I think this makes sense.